static/styles/account/settings.scss

```diff
@@ -30,12 +30,12 @@
 
 .route-account {
   sup {
+    color: #f00;
     font-size: 16px;
-    color: red;
   }
 
   .hint {
+    color: #808080;
```

Comment:
fände es super wenn wir weiterhin alle verwendeten farben in der style.scss als variablen hätten, und wenn es eine sehr spezifische farbe ist dann einfach z.b. $hintColor